extensions/wikia/DesignSystem/i18n/ko.json

```diff
@@ -0,0 +1,19 @@
+{
+  "notifications-no-notifications-message": "No notifications yet.",
```

Comment:
Seems like we shouldn't support a few of these languages if they don't have any other translations, right?